src/corecel/io/LoggerTypes.cc

```diff
@@ -40,7 +40,7 @@ char const* to_color_code(LogLevel lev)
     char c = ' ';
     switch (lev)
     {
-        case LogLevel::debug:      c = 'x'; break;
+        case LogLevel::debug:
```

Comment:
Will this need to be annotated with `[[fallthrough]]` to stop some compilers complaining?